sphinx/builders/html.py

```diff
@@ -365,6 +365,28 @@ def default_translator_class(self):
         else:
             return HTMLTranslator
 
+    @property
+    def math_renderer(self):
+        # type: () -> unicode
+        name = self.get_builder_config('math_renderer', 'html')
```

Comment:
I use `get_builder_config()` to read `html_math_renderer` setting.
It makes math renderers switchable by builder (ex. `epub_math_renderer`)